jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  roots: ['<rootDir>'],
```

Comment:
I believe this is a placeholder?

Reply:
Ah no, Jest substitutes this with the root of the project ;-) But I believe it should be `<rootDir>/src` according to the Jest docs: https://facebook.github.io/jest/docs/en/configuration.html#roots-array-string